wiki/Wiki.md

```diff
@@ -80,3 +80,15 @@ Our wiki consists of the following components:
 If you change the wiki through the web interface at https://github.com/oppia/oppia-android/wiki, the [`wiki.yml`](https://github.com/oppia/oppia-android/blob/develop/.github/workflows/wiki.yml) workflow will be triggered and the wiki will be reset to by pushing the files from the `/wiki` folder to the `oppia/oppia-android.wiki` git repository.
 > **Note**  
 > The `/wiki` folder in the `oppia/oppia-android` repository is the single source of truth for our wiki.
+
+## Tips and Tricks
+
+* ### How can I see markdown (wiki) preview in Android Studio?
```

Comment:
Did you mean to put this header in a bullet list?

Reply:
Done.